src/cdomain/value/cdomains/valueDomain.ml

```diff
@@ -210,6 +211,65 @@ struct
     | TNamed ({ttype=t; _}, _) -> top_value ~varAttr t
     | _ -> Top
 
+  (* top values, except type-based target for addresses *)
```

Comment:
Can this be de-duplicated? E.g. by having a function and supplying it with different `top` functions for addresses?